__tests__/app.test.js

```diff
@@ -120,3 +120,40 @@ describe('GET /api/topics', () => {
     
   });
 
+  describe("GET /api/articles/:article_id/comments", () => {
+    test("200 OK: all comments for an article", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const { comments } = body;
+          expect(comments).toBeInstanceOf(Array);
```

Comment:
Could we also include a length check here? If the response was an empty array, currently this test would auto pass as the forEach would never run.